File: python/dp-e04.py
import string
import random
from sys import argv

def randchar():
    return random.choice(string.letters)

def randstr(length):
    return ''.join([randchar() for x in xrange(length)])

if __name__ == '__main__':
    amount = int(argv[1]) if len(argv) > 1 else 5
    length = int(argv[2]) if len(argv) > 2 else 5

    for x in xrange(amount):
        print(randstr(length))

    

File: javascript/dp-e04.js

function randomItemOf(arr){
  let i = Math.floor(Math.random() * arr.length);
  return arr[i];
}

const letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
const randchar = () => randomItemOf(letters.split(''));

function randstr (length) {
  let result = [];
  for (let i = 0; i < length; i++) {
    result.push(randchar());
  }
  return result.join('');
}

console.log(randstr(parseInt(process.argv[2])));
